static/scripts/rewards/render-transaction/insert-table-data.ts

```diff
@@ -1,6 +1,6 @@
+import { ERC20Permit, ERC721Permit } from "@ubiquibot/permit-generation/types";
```

Comment:
Our coding style guide should have this written as `Erc20Permit`

It is an implicit rule according to the contributing.md in the Ubiquity Dollar repo

Reply:
Got it. We have no linting rule for this case?

Reply:
> We have no linting rule for this case? 

Not yet!